source/bdDetect.py

```diff
@@ -472,6 +472,10 @@ def getConnectedUsbDevicesForDriver(driver: str) -> Iterator[DeviceMatch]:
 		)
 	)
 	for match in usbDevs:
+		if (match.type, match.id) in DeviceMathcer.fallBackDevices:
+			DeviceMathcer.matches.append(match)

```

Comment:
This looks like the `matches` list is ever growing.

Reply:
Okay. I fixed it with clear the mathces list before iteration.

Reply:
Where are you doing that exactly, or wasn't that yet committed?

Reply:
Sorry.  I forgot to push that code.